ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/StandaloneExecutorTest.java

```diff
@@ -306,11 +307,13 @@ public void shouldRunCsasStatements() {
     standaloneExecutor.start();
 
     // Then:
-    verify(engine).execute(csas, ksqlConfig, emptyMap());
+    verify(engine).execute(
+        argThat(arg -> arg.getStatement() instanceof CreateStreamAsSelect),
```

Comment:
Custom matchers in #2432 might be of interest to you for this.

Reply:
Eagerly awaiting that merge!